README.md

```diff
@@ -102,3 +102,52 @@ const InstanceLogger = init(({ name }) => {
 
 const george = InstanceLogger({ name: 'George' }); // Created new instance: George
 ```
+
+
+### overrides()
```

Comment:
I think I misunderstood the implementation you had in mind. Maybe `instanceProps`?